frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -63,9 +64,7 @@ function FileManagerMenu:initGesListener()
 end
 
 function FileManagerMenu:setUpdateItemTable()
-    for _, widget in pairs(self.registered_widgets) do
-        widget:addToMainMenu(self.tab_item_table)
-    end
+    PluginLoader:addToMenu(self.registered_widgets, self.tab_item_table)
```

Comment:
If we believe "Plugins" menu is the right place for all plugins, I agree your suggestion. So far we have several things need to do:
1. PluginLoader is a singleton now, it needs to receive a menu table.
2. All plugins need to be updated.
The disadvantage is it would break the possibility a plugin puts itself to other menu items, even we do not use it. Or in another word, if a plugin needs to do it, it can still use the legacy addToMainMenu() way.

But I doubt whether "Plugins" menu is the right thing to do. A plugin is more like an application of KOReader, it can be placed to the menu group which represents its functionality.

E.g. Read timer, Statistics and Frontlight gesture controller can be placed in text category, or more accurate, a "read experience" group. Storage statistics, Battery statistics and Synchronize time can be placed in a "device" group. Evernote can be placed in "bookmark" group.

We can still keep "plugins" menu in "hamburg" to access all plugins.

What do you think?